perception/pointcloud_octomap_updater/include/moveit/pointcloud_octomap_updater/pointcloud_octomap_updater.h

```diff
@@ -81,7 +81,8 @@ class PointCloudOctomapUpdater : public OccupancyMapUpdater
   std::string point_cloud_topic_;
   double scale_;
   double padding_;
-  double max_range_;
+  double obstacle_range_;
```

Comment:
we should add comments here -- noting that these parameters have the same meaning as the obstacle/voxel layers of the navigation stack -- and that obstacle range is the max range at which an obstacle will be inserted, whereas ray trace range is max distance to ray trace free space.